src/containers/layout/navbar/NavBar.tsx

```diff
@@ -51,21 +62,42 @@ const Navbar = () => {
     openDrawer()
   }
 
+  const findOffersMenu = findOffersChildRoutes.map((childRoute) => (
+    <MenuItem
+      component={Link}
+      key={childRoute.route}
+      onClick={closeMenu}
+      sx={styles.findOfferMenuItem}
+      to={childRoute.path}
+    >
+      {t(`header.${childRoute.route}`)}
+    </MenuItem>
+  ))
+
   const navigationList = navigationItems.map((item, idx, array) => {
     const isLast = array.length - 1 === idx
     const isActive = Boolean(matchPath(item.path, pathname))
 
     return (
       <Fragment key={item.route}>
-        <ListItem>
-          <Typography
-            component={HashLink}
-            sx={styles.navItemText(isActive)}
-            to={item.path}
-          >
-            {t(`header.${item.route}`)}
-          </Typography>
-        </ListItem>
+        {item.route === 'findOffers' ? (
```

Comment:
are we gonna hardcode this 'findOffers', or maybe it would be better to get this from constants?

Reply:
agree, changed